### PR TITLE
Updated Traffic Manager schemas

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

File: schemas/2017-05-01/Microsoft.Network.json [hunk 1]
```diff
@@ -4,306 +4,362 @@
   "title": "Microsoft.Network",
   "description": "Microsoft Network Resource Types",
   "resourceDefinitions": {
-    "trafficManagerProfiles": {
```
Comment (TianoMS) [inline schemas/2017-05-01/Microsoft.Network.json]: >trafficManagerProfiles [](start = 5, length = 22)

why is this changed? I believe the current resource type has the recommended casing.

Comment (EvgenyAgafonchikov) [reply schemas/2017-05-01/Microsoft.Network.json]: All the stuff was autogenerated basing on current Traffic Manager state from azure-rest-api-specs. It looks like all the entries have lower case: [sample](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/trafficmanager/resource-manager/Microsoft.Network/stable/2017-05-01/trafficmanager.json#L18) which is different from [NSG](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/networkSecurityGroup.json#L39) for example.

@dilukose, @KumudD, could you please have a look? If this change is not expected, should we fix schemas manually?

Comment (MikhailTryakhov) [reply schemas/2017-05-01/Microsoft.Network.json]: @TianoMS can you please help to resurrect discussion?

**Comment by EvgenyAgafonchikov** [comment]
closing as outdated
